snakemake/scheduler.py

```diff
@@ -391,7 +392,7 @@ def schedule(self):
                         "Ready jobs ({}):\n\t".format(len(needrun))
                         + "\n\t".join(map(str, needrun))
                     )
-                    run = self.job_selector(needrun)
+                    run = self.job_selector_ilp(needrun)
```

Comment:
Please make this configurable via the command line. `snakemake --scheduler [ilp|greedy]`, with ilp being the default.